test/es-module/test-typescript.mjs

```diff
@@ -336,3 +336,29 @@ test('execute a JavaScript file importing a cjs TypeScript file', async () => {
   match(result.stdout, /Hello, TypeScript!/);
   strictEqual(result.code, 0);
 });
+
+test('expect process.features.typescript to be \'strip\' when --experimental-strip-types', async () => {
+  const result = await spawnPromisified(process.execPath, [
+    '--experimental-strip-types',
+    '-p', 'process.features.typescript',
+  ]);
+
+  match(result.stderr, /Type Stripping is an experimental feature/);
```

Comment:
IMO it makes sense to disable the warning, so if we make a change to the warning it doesn't affect this test.
```suggestion
    '--no-warnings',
    '--experimental-strip-types',
    '-p', 'process.features.typescript',
  ]);

  strictEqual(result.stderr, '');
```